tests/admin_views/tests.py

```diff
@@ -818,6 +818,17 @@ def test_hide_change_password(self):
             msg_prefix='The "change password" link should not be displayed if a user does not have a usable password.'
         )
 
+    def test_template_change_password(self):
+        """
+        Ensure that the auth/user/change_password.html template has
```

Comment:
Just state the expected behavior (no "Ensure that" prefix needed) and include the explanation about password managers. In that case, the ticket doesn't offer any additional info, so we can remove the reference to it.